man/systemd.netdev.xml

```diff
@@ -2179,6 +2182,43 @@
     </variablelist>
   </refsect1>
 
+  <refsect1>
+    <title>[VirtualWLAN] Section Options</title>
+    <para>The [VirtualWLAN] section only applies for virtual WLAN interface, and accepts the following
+    keys:</para>
+
+    <variablelist class='network-directives'>
+      <varlistentry>
+        <term><varname>Phy=</varname></term>
+        <listitem>
+          <para>Specifies the name or index of the WLAN physical layer device. The list of the physical
```

Comment:
I think the "Phy=" name is not very good. Can we call this "PhysicalDevice="? (It can be called "phy" internally in the code, that is OK.)

Also, "Specifies the name or index of the physical WLAN device." ?